lms/djangoapps/ccx/plugins.py

```diff
@@ -31,7 +33,7 @@ def is_enabled(cls, course, user=None):
             # If ccx is not enable do not show ccx coach tab.
             return False
 
-        is_staff_or_instructor = has_access(user, 'staff', course) or has_access(user, 'instructor', course)
+        is_staff_or_instructor = user.has_perm(VIEW_CCX_COACH_DASHBOARD, course.id)
```

Comment:
Do we need to keep the instructor option, too?

Should we pass in the course or the course id?

Reply:
passing course_id will only check staff and instructor access which is what we want here  
not sure why the behavior is different when you pass a course